SeaPublicWebsite/Controllers/FeedbackController.cs

```diff
@@ -34,6 +35,39 @@ public IActionResult FeedbackForm_Post(FeedbackFormViewModel viewModel)
             viewModel.WhatUserToldUs);
         return RedirectToAction(nameof(FeedbackThankYou_Get), "Feedback");
     }
+    
+    [HttpGet("feedback-survey")]
+    public IActionResult FeedbackSurvey_Get()
+    {
+        var viewModel = new FeedbackSurveyViewModel();
+        ViewBag.OnFeedbackPage = true;
+        return View("FeedbackSurvey", viewModel);
+    }
+
+    [HttpPost("feedback-survey")]
+    public IActionResult FeedbackSurvey_Post(FeedbackSurveyViewModel viewModel)
+    {
+        if (!viewModel.VisitReasonList.Any())
+        {
+            ModelState.AddModelError(nameof(viewModel.VisitReasonList), "Please select at least one option");
+        }
+        if (!viewModel.HowInformationHelpedList.Any())
+        {
+            ModelState.AddModelError(nameof(viewModel.HowInformationHelpedList), "Please select at least one option");
+        }
+        if (!viewModel.WhatPlannedToDoList.Any())
+        {
+            ModelState.AddModelError(nameof(viewModel.WhatPlannedToDoList), "Please select at least one option");
+        }
+        if (!ModelState.IsValid)
+        {
+            ViewBag.OnFeedbackPage = true;
```

Comment:
This should be `return FeedbackSurvey_Get()`?
Then you won't need to set the ViewBag either.

Reply:
Can you fix `FeedbackForm_Post` at the same time please?